### PR TITLE
Enhancement/skale 1602 setup tests

### PR DESCRIPTION
**Pull request by badrogger**
Covered almost all skale-node-cli commands

### CODE REVIEW

**Comment by badrogger** [comment]
Finally works

File: core/config.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#   -*- coding: utf-8 -*-
```
Comment (dmitry-tk) [inline core/config.py]: this file is no longer used, plz remove

File: main.py [hunk 2]
```diff
@@ -33,14 +32,15 @@
 
 from core.helper import (login_required, safe_load_texts, local_only,
                          no_node, init_default_logger)
-from configs import CONFIG_FILEPATH, LONG_LINE
+from configs import LONG_LINE
```
Comment (dmitry-tk) [inline main.py]: pull from the develop and update config routes

**Comment by cstrangedk** [comment]
fixes #15

**Pull request merged**